packages/firestore/src/core/firestore_client.ts

```diff
@@ -160,6 +160,14 @@ export class FirestoreClient {
     return persistenceResult.promise;
   }
 
+  /** Enables the network connection and requeues all pending operations. */
+  public enableNetwork(): Promise<void> {
+    return this.asyncQueue.schedule(() => {
+      this.remoteStore.enableNetwork();
```

Comment:
I think we should return enableNetwork() instead of Promise.resolve() ?

Reply:
Yes, that make sense. I also made disableNetwork() return a Promise to align them better.